components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -20,10 +20,10 @@
       </p>
       <h3>NIH Award</h3>
       <p>{{ getSparcAwardNumber }}</p>
-      <template v-if="externalPublications.length">
+      <template v-if="primaryPublication">
         <h3>{{ associatedPublications }}</h3>
```

Comment:
The `associatedPublications` computed property that was removed was for this heading. This can be reverted to `Associated Publication` instead of a computed property.

Reply:
I forgot to change the h3 header, the `associatedPublications` property is no longer needed